pages/common/carbon-now.md

```diff
@@ -0,0 +1,36 @@
+# carbon-now
+
+> Create beautiful images of code.
+> More at <https://github.com/mixn/carbon-now-cli> and <https://carbon.now.sh>.
+
+- Create image from file using default settings:
+
+`carbon-now {{file}}`
+
+- Create image from text in clipboard using default settings:
+
+`carbon-now --from-clipboard`
+
+- Create image standard input using default settings:
+
+`{{input}} | carbon-now`
+
+- Create images interactively for custom settings and optionaly save a preset:
```

Comment:
What exactly does it mean to `optionally` save a preset? Does the command below saves a preset? 
I'm unsure this clearly states the functionality of the command.
```suggestion
- Create images interactively for custom settings and optionally save a preset:
```